README

```diff
@@ -6,3 +6,16 @@ non-root users to view older versions of files and revert changes.
 
 For more information visit http://snapper.io/.
 
+Releasing
+=========
+
+Before releasing package ensure that changes made to package are mentioned
+in `package/snapper.changes`.
+To create package use commands `make -f Makefile.repo && make package`.
+Then use common work-flow to submit package to build service. For factory
+send at first package to devel project YaST:Head on OBS.
+
+When version is increased then git repo have to be tagged, obs project
+filesystems:snapper have to be updated and placing bzip2 on ftp.suse.com.
+When documentation changes e.g. man page or an important functionality, then
+also snapper.io webpages have to be updated.
```

Comment:
git tag is easy but the format has to be right so that a zip can be downloaded, so stick to the already existing convention.

Updating the obs project is not standard since it builds for many distributions, including Debian, so several files have be to edited. Sure, all of that is documented somewhere.

For ftp.suse.com the path should be mentioned (ftp.suse.com/pub/projects/snapper).